app/javascript/lib/honeycrisp.js

```diff
@@ -98,28 +106,32 @@ var followUpQuestion = (function() {
 
                 // set initial state of follow-ups based on the page
                 $(this).find('input').each(function(index, input) {
-                    if($(this).is(':checked') && $(this).attr('data-follow-up') != null) {
-                        $($(this).attr('data-follow-up')).show();
+                    if($(this).attr('data-follow-up') != null) {
+                        $($(this).attr('data-follow-up')).toggle($(this).is(':checked'));
```

Comment:
added this back https://github.com/codeforamerica/vita-min/commit/fa6ea89ab04d87fd7091ddc2f028d7808adcb266#diff-395ae426475fee69171cc850fa90abec0b8c28872fdb039818382a2a40aaa327